babylonclient/msgsender.go

```diff
@@ -142,6 +142,37 @@ func (m *BabylonMsgSender) sendDelegationAsync(stakingTxHash *chainhash.Hash, re
 	}()
 }
 
+func (m *BabylonMsgSender) sendDelExpansionAsync(stakingTxHash *chainhash.Hash, req *sendDelegationRequest) {
```

Comment:
this is almost carbon copy of the `sendDelegationAsync` the only difference is this bit:
```go
txResp, err := m.cl.ExpandDelegation(req.dg)
```
maybe we could generalize this somehow 🤔

Reply:
updated